### PR TITLE
DOCS-5117 Exchange: 2019-05-29 updates

### PR DESCRIPTION
**Pull request by wwarner-salesforce**
DOCS-5117 Exchange: 2019-05-29 updates.

Check with team to confirm that these are not released:
- https://www.mulesoft.org/jira/browse/DOCS-5036 / https://www.mulesoft.org/jira/browse/EXC-3163
- https://www.mulesoft.org/jira/browse/DOCS-5105 / https://www.mulesoft.org/jira/browse/EXC-3637
- https://www.mulesoft.org/jira/browse/DOCS-4979 / https://www.mulesoft.org/jira/browse/EXC-3446

Check text for: https://www.mulesoft.org/jira/browse/EXC-3568

docs-exchange PRs coming soon:
- https://www.mulesoft.org/jira/browse/EXC-3350
- https://www.mulesoft.org/jira/browse/EXC-3636

### CODE REVIEW

File: modules/ROOT/pages/exchange/anypoint-exchange-release-notes.adoc [hunk 1]
```diff
@@ -7,20 +7,29 @@ endif::[]
 
 === New Features
 
+* Improved readability of API instances when requesting API access, especially for long API instance names. (EXC-3636)
 * When using direct upload, provide relevant information about the files that should be uploaded for each asset type. (EXC-3427)
 * Users can now customize the order of the user-generated pages in the public portal. (EXC-3405)
-* Exchange now xref:exchange::to-find-info.adoc[searches] properties defined in RAML and OAS API specifications and the full text of asset portal page content. (EXC-2967)
+* Add SVG image support. (EXC-3350)
```
Comment (mcampo) [inline modules/ROOT/pages/exchange/anypoint-exchange-release-notes.adoc]: Let's specify where the SVG image support was added
```suggestion
* Added support of SVG images for asset portal resources and public portal customization resources. (EXC-3350)
```

File: modules/ROOT/pages/exchange/anypoint-exchange-release-notes.adoc [hunk 2]
```diff
@@ -7,20 +7,29 @@ endif::[]
...
+* Exchange now xref:exchange::to-find-info.adoc[searches] properties defined in RAML and OAS API specifications and the full text of asset portal page content. Users no longer need to create complex tags and categories to enable finding assets, since they can now search the API docs and API specs in addition to name, categories, and tags. (EXC-2967)
 
 === Fixed in this Release
 
+* The "View WSDL" link is now validated. URLs without a protocol default to `http`. (EXC-3641)
```
Comment (mcampo) [inline modules/ROOT/pages/exchange/anypoint-exchange-release-notes.adoc]: Let's specify that validation actually happens in the "New Asset" form. Also, defaulting to `http` for URLs that were entered before the validation is to keep backward compatibility with existing data.
```suggestion
* When creating a SOAP API, the "File URL" link is now validated. File URLs that didn't contain a protocol (`http` / `https`) entered before the validation was in place now default to `http`. (EXC-3641)
```

File: modules/ROOT/pages/exchange/anypoint-exchange-release-notes.adoc [hunk 3]
```diff
@@ -7,20 +7,29 @@ endif::[]
...
 * Users can use the mocking service to try the APIs available in public portals with branded domain names. (EXC-3585)
+* Keep names synchronized across API versions. (EXC-3568)
```
Comment (mcampo) [inline modules/ROOT/pages/exchange/anypoint-exchange-release-notes.adoc]:
```suggestion
* Prevent Exchange Contributors and Exchange Admins from changing API name in the public portal, as it should be a read-only view of the asset. (EXC-3568)
```

File: modules/ROOT/pages/exchange/anypoint-exchange-release-notes.adoc [hunk 4]
```diff
@@ -7,20 +7,29 @@ endif::[]
...
 * When an API version is made public, the visibility indicator in the API version drop-down in the asset portal reflects the correct state. (EXC-3513)
 * Remove orphan connectors when an API asset version is deleted. (EXC-3510)
+* Improve readability of portal page tables with long words. (EXC-3495)
 * Generate the correct mocking service URL for OAS API specifications. (EXC-3478)
+* Custom icons for an asset's portal are now used in the dependencies table. (EXC-3471)
 * Tags are no longer overwritten with the previous asset version's tags when updating the asset name. (EXC-3442)
+* Users can now upload assets using relative file paths in an `anypoint-cli` interactive session. (EXC-3423)
 * Unauthenticated access to portal images is disabled for images uploaded after this update. To disable unauthenticated access to an image uploaded before this update, download it, remove it, and xref:exchange::to-describe-an-asset.adoc#add-an-image[upload it again]. (EXC-3394)
 * Prevent inconsistent asset type across multiple versions of an asset. (EXC-3392)
+* Making a version of an API public now depends on validation of that version and not on validation of more recent versions. (EXC-3336)
```
Comment (mcampo) [inline modules/ROOT/pages/exchange/anypoint-exchange-release-notes.adoc]:
```suggestion
* Making a version of an API public now depends on validation of that version and not on validation of uncompleted versions. (EXC-3336)
```

**Comment by mcampo** [comment]
I can confirm that EXC-3163 (Policy portal), EXC-3637 (Version agnostic properties) and EXC-3446 (API as Products) are not released yet.

File: modules/ROOT/pages/exchange/anypoint-exchange-release-notes.adoc (hunk 1 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/exchange/anypoint-exchange-release-notes.adoc]: Yes, that is better. Changed to "Users can now add SVG images as resources for asset portals and public portal customization."

File: modules/ROOT/pages/exchange/anypoint-exchange-release-notes.adoc (hunk 2 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/exchange/anypoint-exchange-release-notes.adoc]: Much better! Changed to "When creating a SOAP API, the "File URL" link is now validated. File URLs entered without a protocol (`http` / `https`) before the validation was in place now default to `http`."

File: modules/ROOT/pages/exchange/anypoint-exchange-release-notes.adoc (hunk 3 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/exchange/anypoint-exchange-release-notes.adoc]: Much better! Thanks. Changed to "Prevent Exchange contributors and Exchange admins from changing the API name in the public portal, which should be a read-only view of the asset."

File: modules/ROOT/pages/exchange/anypoint-exchange-release-notes.adoc (hunk 4 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/exchange/anypoint-exchange-release-notes.adoc]: Thank you! Fixed.

**Comment by wwarner-salesforce** [comment]
That's perfect, @mcampo , and I made the changes you suggested. Thank you very much! If this version looks ready to you, please go ahead and change your review from "Changes requested" to "Approved."

**Pull request merged**